conf/services/changesetReplacement.json

```diff
@@ -0,0 +1,6 @@
+{
+    "strictBounds": "--strict-bounds",
+    "fullReplacement": "--full-replacement",
+    "disableWaySnapping": "--disable-way-snapping",
+    "disableConflation": "--disable-conflation"
+}
```

Comment:
This is the mapping of advanced options in a request to the command line parameters used in `hoot changeset-derive-replacement`